tests/lib/util/metadata.test.ts

```diff
@@ -0,0 +1,9 @@
+import { transformMBMatchURL } from '@lib/util/metadata';
+
+
+describe('transforming match URLs', () => {
+    it('should prefix MB domain', () => {
+        expect(transformMBMatchURL('release/add'))
+            .toBe('*://*.musicbrainz.org/release/add');
```

Comment:
That would definitely be a good idea. Since this is a bit tricky, I'll reference this comment in a new issue and put my thoughts there, so that it doesn't get lost in this PR.

Reply:
Thanks, I should have done that myself right away. I guess I lean towards leaving comments like this one buried somewhere in the code instead of creating a more traceable issue 🙄

Reply:
Don't beat yourself up over it :)